modules/xfeatures2d/src/beblid.cpp

```diff
@@ -0,0 +1,602 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+// Author: Iago Suarez <iago.suarez.canosa@alumnos.upm.es>
+
+// Implementation of the article:
+//     Iago Suarez, Ghesn Sfeir, Jose M. Buenaposada, and Luis Baumela.
+//     BEBLID: Boosted Efficient Binary Local Image Descriptor.
+//     Pattern Recognition Letters, 133:366–372, 2020.
+
+#include <bitset>
+#include "precomp.hpp"
```

Comment:
> precomp.hpp

should go first.

Reply:
You are right, in fact `<bitset>` is no used so I deleted it.